src/Controller/ApiController.php

```diff
@@ -89,6 +75,10 @@ public function synchroAction(Request $request): \Symfony\Component\HttpFoundati
 
             // Get input data
             $data = json_decode($request->getContent(), true);
+            $force = true;
```

Comment:
**suggestion(non-blocking)**:  This could be replaced with a ternary operator

Reply:
Hi @EstelleMyddleware , yes you are right. Could you change it please ? Thanks.